k9mail/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -413,6 +413,8 @@
   <string name="account_settings_reply_after_quote_summary">Quando respondendo mensagens, o texto original deverá aparecer acima da minha resposta.</string>
   <string name="account_settings_strip_signature_label">Tirar assinaturas em respostas</string>
   <string name="account_settings_strip_signature_summary">Assinaturas serão removidas em mensagens com citação</string>
+  <string name="account_settings_bundle_message_followings_label">Agrupar sequências a mensagens</string>
+  <string name="account_settings_bundle_message_followings_summary">Respostas e encaminhamentos serão guardados na mesma pasta que a mensagem original</string>
```

Comment:
Please only change the default language. Translations are handled via [Transifex](https://www.transifex.com/k-9/k9mail/).